app/views.py

```diff
@@ -72,7 +70,19 @@ def convert_to_readable_time(time):
     return time.strftime('%a %b %d, %Y %I:%M:%S %p')
 
 
-def display(forecast, frequency):
+def convert_to_celsius(temp):
+    return round((temp - 32) * 5 / 9, 2)
+
+
+def convert_to_percentage(amount):
+    return int(amount * 100)
+
+
+def display(forecast):
+    return "Temp: {}C / {}F, Humidity: {}%, Precip: {}%, Summary: {}".format(convert_to_celsius(forecast.temperature), forecast.temperature, convert_to_percentage(forecast.humidity), convert_to_percentage(forecast.precipProbability), forecast.summary)
```

Comment:
That response(format) is what Avner wanted. Regarding issues on temperature units will be on a different PR.